### ISSUE TITLE
SIgnup: If user already exists in user table, and even then tries to signup

### ISSUE THREAD

**Issue by hemangsk** [SDC-USICT/Recruitment_Portal#62]
Then it redirects to verify page, instead there should be a error showing user already exists

**Comment by Foberm** [comment]
I can do this issue

**Comment by hemangsk** [comment]
Hey @Foberm , Thanks for taking this up, ping me / @surender7790 / @rahulcode22 / @sksameer   if you need anything :)
You must have got an email with an invite for collaboration, please accept the invitation so that we can assign this issue to you :)

**Comment by surender7790** [comment]
@hemangsk if the user exists and verified then it should flash a message "User Already Exist. Please Login".
OR if the user exists and not verified then it should redirect to verify page.

**Comment by hemangsk** [comment]
@Foberm  ^^

**Comment by hemangsk** [comment]
ping @Foberm , would you like to take this up?

**Comment by egin10** [comment]
can i help you for hacktoberfest ?

**Comment by hemangsk** [comment]
Hey @egin10 , sure! shall I assign it to you?

**Comment by egin10** [comment]
I'll try to do it, then if it's done i'll tell you and try to make PR.
is this use expressjs ?

**Comment by hemangsk** [comment]
sure sounds great

**Comment by egin10** [comment]
is this issue like this ?

**if the user exists and verified then it should flash a message "User Already Exist. Please Login".
OR if the user exists and not verified then it should redirect to verify page.**

**Comment by hemangsk** [comment]
yeah, let me know if you need any support to get acquainted with the
project. We also have a gitter channel  , its linked in the readme

**Comment by egin10** [comment]
I'll try it

**Comment by egin10** [comment]
may i know what's database you're use for this repo ? MongoDB or MySQL
